src/win32/win32gliface.cpp

```diff
@@ -1141,7 +1141,7 @@ void Win32GLFrameBuffer::ReleaseResources ()
 
 void Win32GLFrameBuffer::SetVSync (bool vsync)
 {
-	if (vsyncfunc != NULL) vsyncfunc(vsync);
+	if (vsyncfunc != NULL) vsyncfunc(vsync ? 1 : 0);
```

Comment:
Strictly speaking no, assuming that all C++ compilers actually map bool to 0 and 1. In any case, I personally consider it a bad practice to pretend something is a bool when it isn't - wglSwapInterval takes an 'swap interval' integer, not a boolean.